index.html

```diff
@@ -35,143 +37,303 @@
-                        <a href="https://stripe.com/docs/api#usage_records" target="_blanks">Usage Records API</a>, and providing customers the option to pay with our new hosted invoices.</a>
+                        Explore our guides and examples to integrate Stripe. Our documentation covers the essentials:
                     </p>
-                    <a href="https://github.com/stripe/stripe-billing-typographic" class="button" target="_blank">Get the code</a>
+                    <ul>
+                        <li><a href="https://stripe.com/docs/payments?payments=popular">Payments</a> - Build an
+                            integration to accept payments online or in person</li>
+                        <li><a href="https://stripe.com/docs/business">Business operations</a> - Monitor, protect, and
+                            report on your money
+                        </li>
+                        <li><a href="https://stripe.com/docs/financial-services">Financial services</a> - Move, control,
+                            and borrow money with Stripe</li>
+                    </ul>
+
+
+                    <a href="https://stripe.com/docs/" class="button" target="_blank">Read
+                        the docs</a>
+
+
                 </div>
+
                 <div class="preview">
-                    <a href="http://rocketrides.io/">
-                        <img src="images/rocket-rides.png" alt="Homepage of Rocket Rides" target="_blank">
+                    <a href="https://github.com/stripe-samples" target="_blank">
+                        <img src="images/samples.png" alt="Stripe samples">
                     </a>
                     <h3 class="title">
-                        <a href="http://rocketrides.io/" class="uppercase" target="_blank">Rocket Rides</a>
+                        <a href="https://github.com/stripe-samples" class="uppercase" target="_blank">Stripe samples</a>
                     </h3>
                     <p>
-                        <a href="http://rocketrides.io/" target="_blank">Rocket Rides</a> is an example marketplace using
-                        <a href="https://stripe.com/connect" target="_blank">Stripe Connect</a>. The demo application shows how you to quickly onboard users to your platform
-                        and let them collect payments. It uses pre-built UI components to get up and running quickly
-                        while still enabling you to customize the user experience.
-                    </p>
-                    <a href="https://github.com/stripe/stripe-connect-rocketrides" class="button" target="_blank">Get the code</a>
+                        A collection of integrations designed to help you get started with your Stripe integration.
+                        Learn how to:
+
+                        <ul>
+                            <li>Quickly <a href="https://stripe.com/docs/checkout/integration-builder">accept one-time
```

Comment:
Should this link to https://github.com/stripe-samples/checkout-one-time-payments instead?